Solutions/Digital Shadows/Data Connectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```diff
@@ -18,22 +18,30 @@ def __init__(self, id, key, secret, url):
         self.id = id
```

Comment:
in `Solutions/Digital Shadows/Data Connectors/Digital shadows/DigitalShadowsSearchlight_API_functionApp.json`

can we fix the capitalisation of `Digital shadows` please?

```
    "sampleQueries": [
        {
            "description" : "All Digital shadows incidents and alerts ordered by time most recently raised",
            "query": "DigitalShadows_CL | order by raised_t desc"
        }
    ],
```

and

```
			{
				"name": "REST API Credentials/permissions",
				"description": "**Digital shadows account ID, secret and key** is required.  See the documentation to learn more about API on the `https://portal-digitalshadows.com/learn/searchlight-api/overview/description`."
			}
```

Also in `readme.md` we have:

```
## Folders

1. `Digital shadows/` - This contains The package...
```

Also note that `The package` should be `the package`